classes/Instructors_List.php

```diff
@@ -204,7 +204,14 @@ public static function update_instructors( $status, $user_ids ): bool {
 			$arr = explode( ',', $user_ids );
 			foreach ( $arr as $instructor_id ) {
 				$instructor_id = (int) sanitize_text_field( $instructor_id );
-				self::remove_instructor_role( $instructor_id, $status );
+				self::instructor_blockage( $instructor_id, $status );
```

Comment:
Remove the status param and pass the param statically from `instructor_blockage` method.